### DISCUSSION TITLE
Pinned Repos, overlap glitch

### DISCUSSION THREAD

**Discussion by BenjaminMichaelis** [community/community#3547] [Mobile] [answered]
When swiping over to the last pinned repo on the mobile profile page, an image of the first pinned repo shows on the right side. This is shown in the following image.

![image](https://user-images.githubusercontent.com/22186029/111114466-831b8a80-8520-11eb-9ed2-c31b40bef94e.png)

**Comment by PranamBhat** [comment]
Hi @BenjaminMichaelis 

Seems like your computer display settings or browser zoom level causing this issue. Check once.

**Reply by BenjaminMichaelis** [reply]
This is on the github mobile app on the iPhone X, so I can't adjust the browser settings, etc.

**Reply by PranamBhat** [reply]
@evi-liu @becca Can you please help @BenjaminMichaelis

**Reply by BenjaminMichaelis** [reply]
It doesn't happen consistently and isn't a restrictive problem as I can access all the pins still so it is not too urgent. Just a graphical issue that I thought github might want to be aware of! :)

**Reply by PranamBhat** [reply]
Let's call UI Developer now 😄 😃

**Reply by BenjaminMichaelis** [reply]
Here you can see when I drag over on the last pin so it moves more left, the static one (which looks the same as the first one) stays static if this helps narrow it down.

![image](https://user-images.githubusercontent.com/22186029/111183444-25615f80-856d-11eb-9812-8dd88e3b7a6a.png)

**Reply by PranamBhat** [reply]
Yup! Got it!

**Reply by nplasterer** [reply]
Thanks for the report. We're looking into it.

**Comment by BenjaminMichaelis** [comment, marked as answer]
@candyho 

I think the original issue here seems like it is still not fixed as of right now so just want to make sure this discussion on this error doesn't get buried as it is a bug it seems like.

In addition, when you are swiping to the repos to the right and you reach the end and swipe to where there are no more repos (the end) it jumps back to the second to last repo or that is repo number (end - 1). This seems like unexpected user behavior as one would expect it to just stay at the last repo and there doesn't seem to be a good reason to move one away in the opposite direction when you are moving to the right

**Reply by BenjaminMichaelis** [reply]
I have a video of the behavior as well, but realized there's not way to attach videos to a discussion thread. Is that feature planned on being added at some point?

**Reply by nplasterer** [reply]
@BenjaminMichaelis are you in our Beta? I believe video uploading for discussions is out in our Beta apps right now.
For Android, users can join our Beta program via [this link](https://play.google.com/apps/testing/com.github.android).
For iOS, users can join our testflight via [this link](https://testflight.apple.com/join/NLskzwi5).

**Reply by BenjaminMichaelis** [reply]
https://user-images.githubusercontent.com/22186029/117324094-557a0000-ae44-11eb-8da7-26a9c4485038.MOV

Video upload working well! Useful feature!
